include/swift/Frontend/Frontend.h

```diff
@@ -467,6 +469,11 @@ class CompilerInstance {
 
   SerializedModuleLoader *getSerializedModuleLoader() const { return SML; }
```

Comment:
It looks like nothing uses this anymore! (That is, `SML` is used within the implementation of CompilerInstance, but the accessor isn't needed.)

It's a little unfortunate we have to track the MemoryBufferLoader explicitly just for this purpose, but we can live with that.